scripts/performance/summarize

```diff
@@ -129,8 +232,13 @@ class Summarizer(object):
 def main():
     parser = argparse.ArgumentParser(usage=__doc__)
     parser.add_argument(
-        '-f', '--benchmark-file', required=True,
-        help='The CSV output file from the benchmark script.')
+        '-f', '--benchmark-files', required=True, nargs='+',
```

Comment:
Yeah I would be fine with this. @JordonPhillips does this affect any the performance work you have been doing if I made that change?